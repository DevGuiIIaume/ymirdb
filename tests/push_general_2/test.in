set a 1
set b a
set c 3
set d c
list entries
set e 5
push e c a
forward a
forward b
forward c
forward d
forward e
backward a
backward b
backward c
backward d
backward e
bye
