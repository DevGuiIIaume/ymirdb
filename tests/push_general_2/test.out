> ok

> ok

> ok

> ok

> d [c]
c [3]
b [a]
a [1]

> ok

> ok

> nil

> a

> nil

> c

> a, c

> b, e

> nil

> d, e

> nil

> nil

> bye
